### PR TITLE
feat: V-JEPA token-grid PoC with autonomous hyperparameter search

### PR DESCRIPTION
**Pull request by c-daly**
## Summary

End-to-end PoC for translating V-JEPA temporal token embeddings into CLIP space, with an autonomous LLM-guided hyperparameter search system.

### Core architecture
- Token-level V-JEPA embeddings `(B, 32, 1024)` — 32 temporal tokens per video, no mean-pooling
- Translator architectures: linear, MLP, residual, transformer, multi-stage pipelines
- Loss functions: MSE, cosine, InfoNCE (false-negative masked), mixed targets
- Both pooled (mean) and token-level training modes

### Autonomous search system
- `search.py`: multi-round search loop with early stopping, convergence patience, resume support
- `coordinator.py`: LLM-guided config generation via gpt-5.4 — analyses results and proposes next experiments
- Targets `txt_R@1` retrieval metric; switches target dynamically via `TARGET:` directive from coordinator
- OOM guard: hard cap `batch_size ≤ 256` (token training multiplies effective batch by K=32)
- Checkpoint saving on best round result; `--resume` for crash recovery

### Key fixes
- Shape fix: transformer architectures output 2D `(B, D)` — else-branch pools `clip_img` to match
- InfoNCE vs contrastive: vanilla contrastive causes training collapse; coordinator steered to `infonce` only
- CLIP matching: no mean-pooling on token-level CLIP targets; per-token loss then pooled for metrics
- Sequential experiment ID prefix `exp_NNN_` enforced for stable leaderboard ordering

### Results so far (MSR-VTT)
- Best `txt_R@1 = 0.371` (target: 0.42), `img_R@5 = 0.944`
- 80+ experiments run across 15+ rounds; token-level configs actively being explored

### Design docs
- `docs/superpowers/plans/2026-03-12-vl-jepa-autonomous-search.md`
- `docs/superpowers/specs/2026-03-12-vjepa-explorer-design.md`
- `docs/superpowers/specs/2026-03-11-vjepa-grounding-integration-design.md`

## Test plan
- [ ] Verify `search.py --resume` loads existing `experiment_log.json` and continues from correct round
- [ ] Run one round manually: `python search.py --rounds 1 --configs-per-round 2 --max-experiments 2`
- [ ] Confirm OOM guard fires for `batch_size > 256`
- [ ] Check `coordinator.py` parses `TARGET:` directive from LLM analysis correctly
- [ ] Validate token-level training path: `num_tokens=32` produces `(B*32, D)` loss, pooled metrics

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,10 +40,15 @@ PoCs/**/*.pdf
 PoCs/**/embedding_cache.json
 PoCs/**/tiny_mind.json
 
+*.h5
+*logs.json
```
[the remark this answers is not in the document]
Comment (c-daly) [reply .gitignore]: Fixed in 6c12dda — added `*.log`, `run_logs/`, and `PoCs/vl_jepa/logs/` to `.gitignore`.
